pennylane/workflow/qnode.py

```diff
@@ -506,7 +519,7 @@ def __init__(
         # Default to "gradient" to maintain default behaviour of "draw" and "specs"
         expansion_strategy = expansion_strategy or "gradient"
 
-        if interface not in SUPPORTED_INTERFACES:
+        if interface not in get_args(SUPPORTED_INTERFACE_USER_INPUT):

```

Comment:
This will still behave correctly since using `in` will only inspect the keys of the dictionary.
```suggestion
        if interface not in SUPPORTED_INTERFACES:
```